ntcore/src/main/native/cpp/ntcore_c.cpp

```diff
@@ -101,6 +103,22 @@ static void DisposeEvent(NT_Event* event) {
   }
 }
 
+static PubSubOptions ConvertToCpp(const NT_PubSubOptions* in) {
+  PubSubOptions out;
+  out.structSize = in->structSize;
```

Comment:
Is `sizeof(NT_PubSubOptions) == sizeof(PubSubOptions)` guaranteed? Is `sizeof(NT_Bool) == sizeof(bool)` guaranteed?

I'm not sure the above invariants are held across all platforms. I think the safest way to do this conversion would be to zero-initialise another `NT_PubSubOptions`, `memcpy`, then assign each non-size field from the copy.

Reply:
Whoops.  I simply shouldn't initialize out.structSize here, it's default-initialized to the correct size.

Reply:
I suppose we don't have to worry about this right now, but we need to make sure we don't read uninitialised memory here in future too.